src/interface/widgets/PlaysWidget.cpp

```diff
@@ -65,12 +65,12 @@ namespace rtt::ai::interface {
 	 *   start:
 	 */
     void PlaysWidget::updatePlays() {
-        QString ss = {};
-        for (auto& each : ApplicationManager::plays) {
-            ss += formatPlay(each.get());
-        }
-        auto sliderPos = verticalScrollBar()->sliderPosition();
-        setHtml(ss);
-        verticalScrollBar()->setSliderPosition(sliderPos);
+//        QString ss = {};
+//        for (auto& each : ApplicationManager::plays) {
```

Comment:
85% sure it's because ApplicationManager::plays isn't mutexed.

Reply:
It's uncommented now and ai does not segfault. We'll keep an eye on it!